app/code/Magento/Checkout/view/frontend/templates/shipping/price.phtml

```diff
@@ -5,7 +5,12 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Shipping\Price */ ?>
+<?php
+/**
+ * @var \Magento\Checkout\Block\Shipping\Price $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+?>
 
```

Comment:
Please remove here not needed empty line
```suggestion
```

Reply:
fixed